src/System.Reflection.Metadata.Cil/tests/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name":"System.Reflection.Metadata.Cil.Tests",
+    "dependencies": {
+        "Microsoft.NETCore.TestHost": "1.0.0-beta-23409",
+        "Microsoft.NETCore.Platforms": "1.0.1-beta-23409",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.0.1-beta-23409",
```

Comment:
It isn't recommended to directly reference the CoreCLR runtime specifically. Instead just reference Microsoft.NETCore.Runtime.